src/app/GitCommands/Git/GitModule.cs

```diff
@@ -316,11 +317,15 @@ public IConfigFileSettings LocalConfigFile
         // 4) branch, tag name, errors, warnings, hints encoded in system default encoding
         public static readonly Encoding LosslessEncoding = Encoding.GetEncoding("ISO-8859-1"); // is any better?
 
-        public Encoding FilesEncoding => ((ConfigFileSettings)EffectiveConfigFile).FilesEncoding ?? new UTF8Encoding(false);
+        private Encoding? _defaultEncoding;
+
+        private Encoding DefaultEncoding => _defaultEncoding ??= new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
```

Comment:
Why is the field needed? Why not init the property straight away? Why not make it static?

Reply:
👍
Yes, lazy instantiation is not worth it.